docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -1,500 +1,594 @@
+   
+   ```code
+   $ tsh logout
+   ```
+
+1. Sign in to your **root cluster** from your administrative workstation using 
+your Teleport username:
+   
+   ```code
+   $ tsh login --proxy=<Var name="rootcluster.teleport.sh" /> --user=<Var name="myuser" />
+   ```
+
+   Replace `rootcluster.teleport.sh` with the Teleport root cluster domain and 
+   `myuser` with your Teleport username.
+
+1. Generate the invitation token by running the following command:
+   
+   ```code
+   $ tctl tokens add --type=trusted_cluster --ttl=5m
+   The cluster invite token: (=presets.tokens.first=)
+   ```
+
+   This command generates a trusted cluster invitation token to allow an inbound 
+   connection from a leaf cluster. The token can be used multiple times. In this
+   command example, the token has an expiration time of five minutes.
+   
+   Note that the invitation token is only used to establish a
+   connection for the first time. Clusters exchange certificates and
+   don't use tokens to re-establish their connection afterward.
+
+   You can copy the token for later use. If you need to display the token again,
+   run the following command against your root cluster:
+
+   ```code
+   $ tctl tokens ls
+   Token                                                    Type            Labels   Expiry Time (UTC)
+   -------------------------------------------------------- --------------- -------- ---------------------------
+   (=presets.tokens.first=)                                 trusted_cluster          28 Apr 22 19:19 UTC (4m48s)
+   ```
+
+1. Create a resource configuration file called `trusted_cluster.yaml` with the
+following content:
 
-<Tabs>
-<TabItem scope={["oss", "enterprise"]} label="Self-Hosted">
-
-```code
-$ curl https://${PROXY?}/webapi/ping | jq .proxy.ssh.public_addr
-"teleport.example.com:443"
-```
-
-</TabItem>
-<TabItem scope={["cloud"]} label="Teleport Enterprise Cloud">
-
-```code
-$ curl https://${PROXY?}/webapi/ping | jq .proxy.ssh.public_addr
-"mytenant.teleport.sh:443"
-```
-
-</TabItem>
-
-</Tabs>
-
-#### `spec.role_map`
-
-When a leaf cluster establishes trust with a root cluster, it needs a way to
-configure access from users in the root cluster. Teleport enables you to limit
-access to Trusted Clusters by mapping Teleport roles to cluster labels.
-
-When creating a `trusted_cluster` resource, the administrator of the leaf
-cluster must define how roles from the root cluster map to roles on the leaf
-cluster.
-
-`trusted_cluster.yaml` uses the following configuration:
-
-```yaml
-  role_map:
-    - remote: "access"
-      local: ["visitor"]
-```
-
-Here, if a user has the `access` role on the root cluster, the leaf cluster will grant
-them the `visitor` role when they attempt to log in to a Node.
-
-If your user on the root cluster has the `access` role, leave this as it is. If
-not, change `access` to one of your user's roles.
+   ```yaml
+   kind: trusted_cluster
+   version: v2
+   metadata:
+     name: rootcluster.teleport.sh
```

Comment:
Similar comment to above. Let's come up with some name to refer to the root cluster, and use it consistently throughout the page. So far we have used both `sso.example.com` and `rootcluster.teleport.sh` to refer to the same cluster.